src/ngrest/base/NgRestModel.php

```diff
@@ -1051,11 +1051,69 @@ public function ngRestConfig($config)
         foreach ($this->ngRestActiveWindows() as $windowConfig) {
             $config->aw->load($windowConfig);
         }
+        // get the scope based config options if no ngRestConfigOptions() are defined
+        $configOptions = !empty($this->ngRestConfigOptions())? $this->ngRestConfigOptions() : $this->getNgRestScopeConfigOptions($config);
```

Comment:
maybe it would be more easy to read when not negate the value:

```
empty($this->ngRestConfigOptions()) ? $this->getNgRestScopeConfigOptions($config) : $this->ngRestConfigOptions();
```

or is there any reason?

Reply:
No, there is no specific reason.. I've changed the code.